src/averageAges.js

```diff
@@ -37,7 +47,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const female = withChildren
+    ? people.filter(({ sex }) => sex === 'f').filter(
+      ({ name }) => people.some(({ mother }) => mother === name))
+    : people.filter(({ sex }) => sex === 'f');
```

Comment:
Do you really need to call filter method three times? 
You can call it once like in calculateMenAverageAge function